docs/src/accelerators.md

```diff
@@ -0,0 +1,100 @@
+# Accelerators
+
+Here, we introduce the Accelerator structs in `Accelerators.jl`, which have been adapted from gradient-based methods in order to accelerate gradient-free ensemble Kalman processes.
+
+While developed for use with ensemble Kalman inversion (EKI), Accelerators have been shown to accelerate convergence for ensemble transform Kalman inversion (ETKI) as well. They are also implemented for use with unscented Kalman inversion (UKI) and the ensemble Kalman sampler (EKS); however, acceleration is experimental for these methods, and often results in instability.
+
+## "Momentum" Acceleration in Gradient Descent
+
+In traditional gradient descent, one iteratively solves for $x^*$, the minimizer of a function $f(x)$, by performing the update step 
+
+```math
+x_{k+1} = x_{k} + \alpha  \nabla f(x_{k}), 
+```
+
+where $\alpha$ is a step size parameter.
+In 1983, Nesterov's momentum method was introduced to accelerate gradient descent. In the modified algorithm, the update step becomes 
+
+```math
+x_{k+1} = x_{k} + \beta (x_{k} - x_{k-1}) + \alpha  \nabla f(x_{k} + \beta (x_{k} - x_{k-1})), 
+```
+
+where $\beta$ is a momentum coefficient. Intuitively, the method mimics a ball gaining speed while rolling down a constantly-sloped hill.
+
+## Implementation in Ensemble Kalman Inversion Algorithm
+
+EKI can be understood as an approximation of gradient descent ([Kovachki and Stuart 2019](https://iopscience.iop.org/article/10.1088/1361-6420/ab1c3a)). This fact inspired the implementation of Nesterov momentum-inspired accelerators in the EKP package. For an overview on the algorithm without modification, please see the documentation for Ensemble Kalman Inversion.
```

Comment:
As a better story line for this paragraph, maybe say that 
(1) Nesterov can be used to accelerate Gradient flows (e.g. Su et al. 2016, https://arxiv.org/abs/1503.01243)
(2) EKI approximates "a form of gradient flow". use reference https://arxiv.org/abs/2209.11371 to talk about this rather than Kovachki stuart. 
(3) Kovachki stuart 2019 used a particle-based modification of the Nesterov acceleration and saw acceleration. 
(4) We use this update to accelerate a variety of our processes

Reply:
Hi! I just pushed a modified version with this story line, but I am not confident that the section flows well. Please let me know if there's anything else I can do to make it clearer. Thanks!